modules/distribution/resources/operation_policies/specifications/ccChangeHTTPMethod_v1.json

```diff
@@ -5,15 +5,6 @@
   "displayName": "Change HTTP Method",
   "description": "This policy allows you to change the HTTP Method",
   "policyAttributes": [
-    {
-      "name": "currentMethod",
-      "displayName": "Current HTTP Method",
-      "description": "Method of the HTTP request to be updated",
-      "validationRegex": "^(GET|PUT|POST|PATCH|DELETE|HEAD|OPTIONS)$",
-      "type": "Enum",
-      "allowedValues" : ["GET","PUT","POST","PATCH","DELETE","HEAD","OPTIONS"],
-      "required": true
-    },
     {
```

Comment:
I guess we have versions in the API policy as well. I am not sure whether we can simply change this or we should create v2 of the same policy.

@1akshitha

Reply:
Policy version support is still not visible in the UI and is not included in the 4.2.0 release.

Reply:
we have added the version support. @CrowleyRajapakse have we added the version support to master branch? If so, we can add it as a v2.